src/points.jl

```diff
@@ -30,6 +31,7 @@ immutable LatLon{T <: Number}
     lat::T
     lon::T
 end
+LatLon(lat, lon) = LatLon(promote(lat, lon)...)
```

Comment:
Not at all, thanks for pointing it out.  I only found it by accident when fixing some other things.